scripts_src/global/gl_k_alcohl.ssl

```diff
@@ -136,12 +154,15 @@ procedure start begin
     end
 
     // relief
-    if ((game_time - timer_relief) >= TIME_ONE_WEEK) then begin
+    if ((game_time - time_of_addiction) >= DURATION_OF_ADDICTION) then begin
         set_global_var(GVAR_ALCOHOL_ADDICT, 0);
         set_critter_stat(dude_obj, STAT_ag, 1);
         set_critter_stat(dude_obj, STAT_ch, 1);
         display_mymsg(MSG_relieved);
+        display_msg(string_format (message_str_game(GAME_MSG_ITEM, 1), 1, message_str_game(GAME_MSG_STAT, 105) ) ); // "You gained 1 Agility"
```

Comment:
Think won't work at all. You presume English, while in other languages phrase structure can be completely different.
Additionally, separate messages seem unnecessary and just pollute PDA screen. I would combine then.
Also, for clarity I would say "regained", rather than "gained".

Reply:
Fair enough, I just tried to re-use already existing strings, since I reasoned, I at least wouldn't introduce new problems. I guess there will be similar problems once someone tries to translate the strings used by the engine directly, but no need to solve everything at once.

I just put the strings into the msg file then.